models/repo_pushmirror.go

```diff
@@ -0,0 +1,106 @@
+// Copyright 2021 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package models
+
+import (
+	"errors"
+	"time"
+
+	"code.gitea.io/gitea/modules/log"
+	"code.gitea.io/gitea/modules/timeutil"
+
+	"xorm.io/xorm"
+)
+
+var (
+	// ErrPushMirrorNotExist mirror does not exist error
+	ErrPushMirrorNotExist = errors.New("PushMirror does not exist")
```

Comment:
NOTE: other error types for models package are declared in models/error.go ... throu I don't know if we should declare them in there used files instead ...

this is just a smal code inconsistency

Reply:
I would vote for a location near/in the used file and not a god-file with all errors. Anyway in this case it's not a new error "type". If you search for `= errors.New` in the code they appear all in the file which uses them.